code/src/App.js

```diff
@@ -1,9 +1,37 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react';
+
+import { ThoughtList } from 'components/ThoughtList';
+import { ThoughtsInput } from './components/ThoughtsInput';
+import { THOUGHTS_URL } from './urls';
 
 export const App = () => {
+  const [thoughts, setThoughts] = useState([]);
+
+  const fetchThoughts = () => {
+    fetch(THOUGHTS_URL)
+      .then((res) => res.json())
+      .then((thought) => setThoughts(thought))
+      .catch((error) => console.error(error));
+  }
+
+  const postThoughts = (newThought) => {
```

Comment:
You don't need () when passing only one argument here. On the other hand the code almost look's better with (newThought) then newThought :D

Reply:
Tried to remove it but then it wouldn't compile :(